src/pointerAction.js

```diff
@@ -350,11 +350,16 @@ export function dndzone(node, options) {
         window.removeEventListener("mouseup", handleFalseAlarm);
         window.removeEventListener("touchend", handleFalseAlarm);
     }
-    function handleFalseAlarm() {
+    function handleFalseAlarm(e) {
         removeMaybeListeners();
         originalDragTarget = undefined;
         dragStartMousePosition = undefined;
         currentMousePosition = undefined;
+
+        // dragging initiated by touch events prevents onclick from initially firing
+        if (e.type === "touchend") {
```

Comment:
cool solution but preventDefault is running regardless of whether it is a touch event or not, right? if so, this will not allow clicking with a mouse.

Reply:
I missed that you said you tested it with a mouse and the preventDefault doesn't stop the click if i understand you correctly. if that's the case you can ignore this comment. I can try to find time to test this properly tomorrow and if it works and the package.json and release-notes are updated i will merge